starter-code/css/main.css

```diff
@@ -6,3 +6,112 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  font-family: Helvetica,Arial,sans-serif;
+  font-weight: lighter;
+  margin: 0;
+}
+
+/*HEADER SECTION*/
+
+header {
+  padding: 10px 40px 0px 40px;
+}
+
+header img {
+  width: 14%;
+}
+
+ul {
+  display: inline-block;
+  float: right;
+  list-style-type: none;
+}
+
+li {
+  display: inline-block;
+  padding-left: 50px;
+  font-size: 25px;
+}
+
+/*SECTION ONE*/
+
+.music {
+position: relative;
+}
+
+.texte {
+position: absolute;
```

Comment:
It works really well in your case but I'm sure there are simpler way to do what you want without a `position: absolute`.